include/sys/zio_compress.h

```diff
@@ -157,12 +157,6 @@ extern zio_compress_info_t zio_compress_table[ZIO_COMPRESS_FUNCTIONS];
 extern void lz4_init(void);
 extern void lz4_fini(void);
 
-/*
- * specific to zstd user space implementation only
- */
-extern void zstd_fini(void);
-extern int zstd_init(void);
```

Comment:
@BrainSlayer This is the removal you refered to, concerning userspace loading, right?

Reply:
no. its in spa_misc.c

Reply:
@c0d3z3r0 This was actually wrong and correctly deleted.